docs/proposals/scc-check.md

```diff
@@ -0,0 +1,89 @@
+## Problem
+It is impossible for a user to know whether the `PodSpec` he's describing will actually be allowed by the current SCC rules.
+This becomes even more problematic when trying to reason about whether a `DeploymentConfig` or `ReplicationController` or other resource containing a `PodSpec` 
+that a system component will eventually create will work.
```

Comment:
In the future when we have dry-run of admission is this obviated?

Is this specific to PodSecurityPolicy?  Or generic and we'd hook other things up?